lib/services/elasticsearch.js

```diff
@@ -309,20 +308,18 @@ class ElasticSearch extends Service {
    *
    * @param {String} index - Index name
    * @param {String} collection - Collection name
-   * @param {Object} query - Query to match
+   * @param {Object} searchBody - Search request body (query, sort, etc.)
    *
-   * @returns {Promise.<Object>} { count }
+   * @returns {Promise.<Number>} count
    */
-  count (index, collection, query = {}) {
+  count (index, collection, searchBody = {}) {
     const esRequest = {
       index: this._getESIndex(index, collection),
-      body: query
+      body: searchBody
     };
 
     return this.client.count(esRequest)
-      .then(({ body }) => ({
-        count: body.count
-      }))
+      .then(({ body }) => body.count)
```

Comment:
Breaking changes on the API should be postponed to Kuzzle v3.
You should either revert this change, or modify the value returned by `collectionController.count` to prevent a breaking change, which would force us to release a new major version of all SDks.

Reply:
Yes, I agree. I will modify the collectionController accordingly in the subsequent PR.  
For each method, I read the doc first to ensure that I'm not making a breaking change and then I explicitly write the response object, even if the one returned by the Elasticsearch service is already well formatted

Reply:
Right. Could you add a `@todo` to make sure we don't forget?